webapp/components/form/Select/Select.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
I'm sure you can get rid of that eslint-disable now

Reply:
Thanks